build.sbt

```diff
@@ -12,6 +12,10 @@ ThisBuild / developers := List(
 )
 ThisBuild / startYear := Some(2022)
 
+ThisBuild / tlCiReleaseStepSummaryTableInfo += (
+  "Scala Versions" -> crossScalaVersions.value.toString
+)
```

Comment:
@armanbilge does it makes sense to keep it for this project? It's an sbt plugin after all...

Reply:
Eh, we have a few things like this. dog-fooding various features within sbt-typelevel itself is a great way to test them. Esp since we don't have any real tests as you well know 😜